docs/Guide/plugins/plugin-api.mdx

```diff
@@ -2,4 +2,83 @@
 title: Plugin API
 ---
 
-## TODO
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+This plugin provides an API endpoint for your bot that can be called from external services. A good exemplary use case
+for this is once your bot grows to have enough configuration options that you want to offer a website to your end-users
+to change those settings, and your website needs to interface with the bot for this to work.
+
+## Installation
+
+```bash npm2yarn2pnpm
+npm install @sapphire/plugin-api @sapphire/framework discord.js discord-api-types
+```
+
+## Usage
+
+Start by importing the registration file in your project to use the plugin:
+
+<Tabs groupId="language-choice">
+
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript
+require('@sapphire/plugin-api/register');
+```
+
+</TabItem>
+
+<TabItem value="esm" label="ESM" default>
+
+```javascript
+import '@sapphire/plugin-api/register';
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="TypeScript" default>
+
+```typescript
+import '@sapphire/plugin-api/register';
+```
+
+</TabItem>
+</Tabs>
+
+Then, you can use the following configuration options in your SapphireClient extension class or initializer. This will
+either be located in your `new SapphireClient` constructor call, or `super` in your constructor method if you use an
+extension class.
+
+```javascript
+{
+  auth: {
+	// The application/client ID of your bot.
```

Comment:
This is a general comment but periods at the end of comments or lack thereof is not consistent throughout the guides & the documentation and it makes me a bit mad sometimes :')

Reply:
i cannot understand but can you explain?

Reply:
Throughout the docs and guides, sometimes comments end with periods, and sometimes they don't. Just a little consistency thing.